### ISSUE TITLE
Enabling "IndividualProgression.QuestXPFix " does not honor "Rate.XP.Quest" Wordserver Settings. 

### ISSUE THREAD

**Issue by XSNomad** [ZhengPeiRu21/mod-individual-progression#7]
There is also a display bug 
![image](https://user-images.githubusercontent.com/60951221/187593911-95f834e6-c8c5-4eb0-bae3-a2fb150835fe.png)
![image](https://user-images.githubusercontent.com/60951221/187593928-c7dccac3-2f89-4898-88fa-e517ee89c662.png)

**Comment by ZhengPeiRu21** [comment]
Thank you, I did not consider quest XP rate. I will change it to honor that modifier and look into the display issue.

**Comment by ZhengPeiRu21** [comment]
Quest XP Rate should now be working after commit: https://github.com/ZhengPeiRu21/mod-individual-progression/commit/72cbe4fdf9ba9cfb8754e73fb0f9a7ccd523e7ca

Showing correct quest XP will be fixed once this PR is merged into AzerothCore: https://github.com/azerothcore/azerothcore-wotlk/pull/12919

**Comment by ZhengPeiRu21** [comment]
AzerothCore PR has been merged, this should now working as intended.
